lib/ansible/module_utils/network/iosxr/iosxr.py

```diff
@@ -415,7 +416,15 @@ def load_config(module, command_filter, commit=False, replace=False,
         if module._diff:
             diff = get_config_diff(module)
 
-        if commit:
+        if replace:
+            cmd = {'command': 'commit replace',
+                   'prompt': 'This commit will replace or remove the entire running configuration',
+                   'answer': 'yes'}
+            conn.edit_config(cmd)
+            # TODO: add a delay if commit replace operation takes too long
+            # sleep(5)
+            conn.edit_config('end')
```

Comment:
cmd can be a list and commands can be sent as a list to avoid invoking `edit_config` twice`?